modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/life.dm

```diff
@@ -32,6 +32,7 @@
 	RegisterSignal(src, COMSIG_BLOODSUCKER_ON_LIFETICK, PROC_REF(HandleDeath))
 
 /datum/antagonist/bloodsucker/proc/on_revive(mob/living/source)
+	SIGNAL_HANDLER
```

Comment:
These only need to exist in the actual signal, you can remove them from here

Reply:
What do you mean actual signal? SIGNAL_HANDLER needs to be in any proc that's registered to a signal

Reply:
Oh then I must've missed on_revive being called by a signal/registered as such, sorry about that

Reply:
Would be nice then if you un-requested changes if you can figure out ow